### PR TITLE
Task 3: Pytest for wrapper classes part 1

### PR DESCRIPTION
**Pull request by Rahul-JOON**
This PR contains the test case for the wrapper class - ExternalContact as mentioned in #99 

Because the codebase is not broken, all of the test cases are currently passing.

### CODE REVIEW

File: tests/test_contact_wrapper_classes.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        ]
+    )
+
+
+MockRod = type("MockRod", (object,), {"__init__": mock_rod_init})
+
+MockRigidBody = type("MockRigidBody", (object,), {"__init__": mock_rigid_body_init})
+
+
+def test_external_contact_with_collision():
```
Comment (armantekinalp) [inline tests/test_contact_wrapper_classes.py:78]:
```suggestion
def test_external_contact_rod_rigid_body_with_collision():
```

File: tests/test_contact_wrapper_classes.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+    assert_allclose(
+        mock_rigid_body.external_torques,
+        np.array([[-2.092858], [0.245406], [0.310859]]),
+        rtol=tol,
+        atol=tol,
+    )
+
+
+def test_external_contact_without_collision():
```
Comment (armantekinalp) [inline tests/test_contact_wrapper_classes.py:116]:
```suggestion
def test_external_contact_rod_rigid_body_without_collision():
```

File: tests/test_contact_wrapper_classes.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    ext_contact = ExternalContact(k=1.0, nu=0.5)
+    mock_rigid_body.position_collection = np.array([[400], [500], [600]])
+    ext_contact.apply_forces(mock_rod, 0, mock_rigid_body, 1)
+
+    assert_allclose(mock_rod.external_forces, mock_rod.external_forces)
+    assert_allclose(mock_rigid_body.external_forces, mock_rigid_body.external_forces)
+    assert_allclose(mock_rigid_body.external_torques, mock_rigid_body.external_torques)
+
+
+def test_external_contact_with_two_rods():
```
Comment (armantekinalp) [inline tests/test_contact_wrapper_classes.py:131]:
```suggestion
def test_external_contact_two_rods_with_collision():
```

**Review by armantekinalp — CHANGES_REQUESTED** [review]
Can we add one more test for two rods without collision

**Comment by Rahul-JOON** [comment]
Sure thing! I will add that test.

**Review by armantekinalp — APPROVED** [review]
LGTM thanks

**Review by bhosale2 — APPROVED** [review]
LGTM

**Pull request merged**